includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -116,7 +116,7 @@ public static function onGetTemplateDom( $title, $args, $frame, &$outputText ) {
 	 * @return bool
 	 */
 	public static function onEndBraceSubstitution( $templateTitle, &$templateWikitext, &$parser ) {
-		global $wgEnableContextLinkTemplateParsing, $wgEnableInfoIconTemplateParsing;
+		global $wgEnableContextLinkTemplateParsing, $wgEnableInfoIconTemplateParsing, $wgEnableNavigationTemplateParsing;
```

Comment:
This looks partially redundant with https://github.com/Wikia/app/blob/dev/includes/parser/Parser.php#L3761-L3766, plus in the linked lines there's no reference to $wgEnableNavigationTemplateParsing (and should be if we want to have checks in both places)